### ISSUE TITLE
Null fileKey() in Windows

### ISSUE THREAD

**Issue by kiranclarivate** [awslabs/amazon-kinesis-agent#110]
Hi,
I am running kinesis agent in Windows and it gives out null fileKey().Is there anyway we can use agent on windows.

Thanks,
Kiran

**Comment by benalta** [comment]
Hi, are you still having this issue?

**Comment by kiranclarivate** [comment]
Yes.Actually I have a workaround for this issue.Instead of fileKey() I am using other file attributes to distinguish the unique files. 

Thanks,
Kiran

**Comment by vijayamirtharajingit** [comment]
Now we have Kinesis Agent for Windows to make life bit easier. 
https://docs.aws.amazon.com/kinesis-agent-windows/latest/userguide/what-is-kinesis-agent-windows.html
Happy Logging!
